util/dvsim/SimCfg.py

```diff
@@ -464,15 +473,19 @@ def gen_results_sub(items, results, fail_msgs):
                 regr_results=regr_results,
                 map_full_testplan=self.map_full_testplan)
             results_str += "\n"
-            self.final_total = self.testplan.final_total
-            # append link
-            self.final_total.append(self.append_result_link("Link"))
+            self.results_summary = self.testplan.results_summary
 
         # Append coverage results of coverage was enabled.
         if self.cov and self.cov_report_deploy.status == "P":
             results_str += "\n## Coverage Results\n"
             results_str += "\n### [Coverage Dashboard](cov_report/dashboard.html)\n\n"
             results_str += self.cov_report_deploy.cov_results
+            self.results_summary["Coverage"] = self.cov_report_deploy.cov_total
+        else:
+            self.results_summary["Coverage"] = "N.A."
+
+        # append link
+        self.results_summary["Detail"] = self.append_result_link("Link")
```

Comment:
Move this to `gen_results_summary()`. Also. WDYT about removing the 'Detail' column and using the 'Name' column for the full results?

Reply:
Agree. Updated, thanks